.github/workflows/ci.yml

```diff
@@ -48,14 +48,59 @@ jobs:
       - name: Run database migrations
         run: mix ecto.migrate --prefix realtime
 
-      # - name: Compile with warnings
-      #   run: mix compile --warnings-as-errors
+      - name: Run tests
+        run: mix test
+  mix-format:
+    name: Format ${{ matrix.otp }} / Elixir ${{ matrix.elixir }}
 
-      # - name: Run formatter
-      #   run: mix format --check-formatted
+    defaults:
+      run:
+        working-directory: ./server
 
-      # - name: Run Credo
-      #   run: mix credo
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
```

Comment:
I'm not qualified to comment on the elixir specifics, but given that we're only supporting a single target for each of these vars, getting rid of the matrix would simplify things. I don't think there's a reason to need multiple versions in the future either.